esp/src/yarn.lock

```diff
@@ -0,0 +1,1213 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
Yes - it ensures that all dependencies of dependencies are "locked" at a known versions - even if their semantic version dependencies allow for patch / minor updates.

IOW it ensure that if we gran all the dependencies in the future, they will match.